python/foglamp/common/service_record.py

```diff
@@ -41,7 +53,8 @@ def __init__(self, s_id, s_name, s_type, s_protocol, s_address, s_port, m_port):
         if s_port is not None:
             self._port = int(s_port)
         self._management_port = int(m_port)
-        self._status = 1
+        self._status = ServiceRecord.Status.Running
```

Comment:
even for inner class? is it not accessible as `Status.Running` itself

Reply:
Nah. It is not accessible via Status.Running. It has to be ServiceRecoord.Status.Running.